widget/entry_test.go

```diff
@@ -383,7 +383,14 @@ func TestEntry_PasteFromClipboard(t *testing.T) {
 }
 
 func TestEntry_TappedSecondary(t *testing.T) {
+	// fresh app for this test
+	test.NewApp()
```

Comment:
Is it not sufficient to use test.NewCanvas, then set the content - and then you don't have to look it up to resize either...?

Reply:
No, because `fyne.CurrentApp().Driver().CanvasForObject(…)` is used to determine the canvas for the overlay.
Did I mention that global state objects (like `fyne.CurrentApp()`) are a sure way to hell?

Reply:
This is a fair comment. In the single instance of Application it should not be an issue (outside of test) as only a single app is allowed per process